docs/content/latest/integrations/prisma.md

```diff
@@ -306,27 +308,27 @@ node index.js
 
 ## Clean up
 
-Now that you're done with this exploration, you can clean up the pieces for your next adventure.
+Now that you have completed with this exploration, you can clean up the pieces as follows:
 
-1. Stop the YugabyteDB cluster
+Stop the YugabyteDB cluster:
 
 ```sh
 ./bin/yb-ctl stop
 ```
 
-To completely remove all YugabyteDB data/cluster-state you can instead run:
+To completely remove all YugabyteDB data/cluster-state, you can instead run the following command:
 
 ```sh
 ./bin/yb-ctl destroy
 2. Stop The Prisma container
 docker stop <container-id>
 ```
```

Comment:
This ain't right...

Reply:
Good catch, it looks like this file was messed up a bit.
Fixed.